### PR TITLE
Add command argument modules to Fuseki Main from a Fuseki modules

### LINKED ISSUE #1522: Allow Fuseki Modules to add to the command line argument (Fuseki Main).
### Version

4.6.1

### Feature

Currently, the command line to Fuseki Main is fixed.

The new feature is to allow Fuseki Modules to manipulate (maybe only add to) the command line so a module can get arguments from start-up.

(current static assets are built-in with argument `--base` - this is a theoretical example)
An example would be a `FMod_Content` that needs a directory name to add a static content handler to server as it's built.

Another example would be a `FMod_admin` which would add an argument to provide a user/password combination for the server admin operations.

### Are you interested in contributing a solution yourself?

Yes

### PR DESCRIPTION
**Pull request by afs**
GitHub issue resolved #1522 

Pull request Description:

Allow plugin code to add argument modules to the Fuseki command line.

Includes various clean-up that got done while on this branch.

----

 - [ ] Tests are included.
 - [x] Commits have been squashed to remove intermediate development commit messages.
 - [x] Key commit messages start with the issue number (GH-xxxx or JENA-xxxx)

By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the [Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).

----

See the [Apache Jena "Contributing" guide](https://github.com/apache/jena/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: jena-cmds/src/main/java/org/apache/jena/cmd/Invoke.java [hunk 1]
```diff
@@ -20,10 +20,12 @@
...
 import java.lang.reflect.Method;
-/** Helper code to invoke a main method buy reflection
- *  This means command can be invoked from this module even
- *  if their implementation resides in a module later
- *  in the build sequence.
+
+/**
+ * Helper code to invoke a main method by reflection.
+ * <p>
+ * This means command can be invoked from this module even if the implementation
```
Comment (kinow) [inline jena-cmds/src/main/java/org/apache/jena/cmd/Invoke.java:27]: s/means command can/means the command can/?

File: jena-fuseki2/jena-fuseki-main/src/main/java/fuseki/run.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * limitations under the License.
+ */
+
+package fuseki;
+
+/**
+ * Run FusekiMain.
+ */
+public class run {
+    // This class must not depend in any Jena code (it would trigger
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-main/src/main/java/fuseki/run.java:25]: s/in any/on any/?

**Review by kinow — APPROVED** [review]
:+1:

**Comment by afs** [comment]
Edits fixed.

Also, the PR has been rebased with the POM changes elsewhere.

**Pull request merged**